src/main/java/EntropySolver.java

```diff
@@ -0,0 +1,129 @@
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static com.google.common.collect.Maps.newHashMap;
+import static com.google.common.collect.Maps.newLinkedHashMap;
+import static com.google.common.collect.Sets.newHashSet;
+import static java.lang.Math.log;
+
+// https://gregcameron.com/infinite-wordle/
+
+@Slf4j
+public class EntropySolver {
+    private static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+    private static final Set<String> ACCEPTED = newHashSet();
+
+    public static void main(String[] args) {
+        Set<String> solutions = newHashSet();
+        processInput(solutions);
+
+        // precomputed best at first step
+        log.info("Best Word: {}", "soare");
```

Comment:
Green - If not wanting to compute the first best word at runtime, this should be extracted into a constant rather than a repeated magic value.